assets/js/googlesitekit/components-gm2/ProgressBar.js

```diff
@@ -21,17 +21,50 @@
  */
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
+import {
+	BREAKPOINT_DESKTOP,
+	BREAKPOINT_SMALL,
+	BREAKPOINT_TABLET,
+	BREAKPOINT_XLARGE,
+	useBreakpoint,
+} from '../../hooks/useBreakpoint';
 
 export default function ProgressBar( {
 	className,
 	small,
 	compress,
 	indeterminate,
 	height,
+	smallHeight,
+	tabletHeight,
+	desktopHeight,
 	progress,
 } ) {
+	const breakpoint = useBreakpoint();
+
+	let progressBarHeight = height;
+
+	if ( BREAKPOINT_SMALL === breakpoint && smallHeight ) {
+		progressBarHeight = smallHeight;
+	}
+
+	if ( BREAKPOINT_TABLET === breakpoint && tabletHeight ) {
+		progressBarHeight = tabletHeight;
+	}
+
+	if (
+		( BREAKPOINT_XLARGE === breakpoint ||
+			BREAKPOINT_DESKTOP === breakpoint ) &&
+		desktopHeight
+	) {
+		progressBarHeight = desktopHeight;
+	}
+
 	const margin =
-		typeof height !== 'undefined' ? Math.round( height / 2 ) : undefined;
+		typeof progressBarHeight !== 'undefined'
```

Comment:
Hmm, I know this is not a change introduced by this PR, but I don't see any reason to use this `typeof...` check here, where `progressBarHeight !== undefined` would be a bit more straightforward.

I'm only mentioning this because I suggested adding some more checks for `undefined` above, so it would be nice to be consistent within the file unless there's a clear reason to differ.